web/pandas/_templates/layout.html

```diff
@@ -1,13 +1,7 @@
 <!DOCTYPE html>
 <html>
     <head>
-        <script async="async" src="https://www.googletagmanager.com/gtag/js?id=G-5RE31C1RNW"></script>
-        <script>
-            window.dataLayer = window.dataLayer || [];
-            function gtag(){ dataLayer.push(arguments); }
-            gtag('js', new Date());
-            gtag('config', 'G-5RE31C1RNW');
-        </script>
+        <script defer data-domain="pandas.pydata.org" src="https://views.scientific-python.org/js/script.js"></script>
```

Comment:
Is it necessary to have this snippet in addition to the theme's analytics?

Reply:
Yes, this is for the website, which uses a different tool/theme/layout compared to the docs (which uses pydata-sphinx-theme)